kyvernitis.h

```diff
@@ -39,6 +39,11 @@ struct dc_motor {
 	const struct gpio_dt_spec input_2;
 };
 
+struct mother_msg {
+	uint8_t type;
+	uint32_t crc;
+};
+
```

Comment:
need to decide on the struct that works with both drive as well as other peripherals